src/deepqmc/wf/paulinet/backflow.py

```diff
@@ -1,38 +1,26 @@
 import torch
 from torch import nn
 
-from deepqmc.torchext import SSP, get_log_dnn, idx_perm
-
-__all__ = ()
-
-
-def backflow_cutoff(r, L=0.5):
-    r = r / L
-    return torch.where(r < L, r ** 2 * (6 - 8 * r + 3 * r ** 2), r.new_ones(1))
+from deepqmc.torchext import SSP, get_log_dnn
 
 
 class Backflow(nn.Module):
-    def __init__(self, mol, embedding_dim, subnets_factory=None):
-        if not subnets_factory:
-
-            def subnets_factory(embedding_dim):
-                return (
-                    get_log_dnn(embedding_dim, 1, SSP, n_layers=3),
-                    get_log_dnn(embedding_dim, len(mol), SSP, n_layers=3),
-                )
-
+    def __init__(
```

Comment:
I would actually move this to `omni.py`, because it's a really small class and is only used there. Also, this backflow is entirely different from the one that was here before.